python/lsst/ts/mtaos/mtaos.py

```diff
@@ -980,13 +986,19 @@ async def run_closed_loop(self):
             "MTRotator",
             readonly=True,
             include=["summaryState", "rotation"],
-        ) as mtrotator:
+        ) as mtrotator, salobj.Remote(
```

Comment:
Before you start the `Remotes`, I suggest you reset the value of current rotation and current elevation, just to avoid processing old data, e.g.:

```
self.current_rotator_position = None
self.current_elevation_position = None
```